### PR TITLE
Add TWC 2023 - Registrations Now Open news post

### PR DESCRIPTION
**Pull request by LeoFLT**
PRing this a few hours ahead of schedule for reviewing.

## To-do before merging

- [x] create tournament listing for TWC 2023 (it should have the id = 37)
- [x] create forum thread (set as private until the post is live)
- [x] create Open Collective event

### CODE REVIEW

File: news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "osu!taiko World Cup 2023: Registrations Now Open!"
+date: 2023-02-16 08:00:00 +0000
+---
+
+Bring along your favorite percussion instrument as we once again drum to the beat. The osu!taiko World Cup is back for its 13th instalment!
```
Comment (Walavouchey) [inline news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md:7]:
```suggestion
Bring along your favourite percussion instrument as we once again drum to the beat. The osu!taiko World Cup is back for its 13th instalment!
```

File: news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+![](/wiki/Tournaments/TWC/2023/img/twc2023-banner.jpg)
+
+The TWC has a reputation among its managers as a testing grounds of sorts as far as the official tournaments go, and this year couldn't be any different – we are bringing back the 3 versus 3 format, which made its last appearance in 2017, in the hopes that it shakes things up a bit (looking at you, Japan). This relatively small change impacts many rules, so make sure to get up to speed by reading the [wiki article](/wiki/Tournaments/TWC/2023#ruleset).
```
Comment (Walavouchey) [inline news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md:11]:
```suggestion
The TWC has a reputation among its managers as being a testing ground of sorts as far as the official tournaments go, and this year couldn't be any different – we are bringing back the 3 vs. 3 format, which made its last appearance in 2017, in the hopes that it'd shake things up a bit (looking at you, Japan). This relatively small change impacts many rules, so make sure to get up to speed by reading the [wiki article](/wiki/Tournaments/TWC/2023#ruleset).
```

File: news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Continuing our ongoing efforts towards diversifying the ways the community can contribute to the World Cups, we have created an **[Open Collective](https://opencollective.com/osu-tournaments/events/osu-taiko-world-cup-2023-13130e56)** to allow for donations, which will be added to the prize pool at the end of the competition.
+
+If you're still not familiar with Open Collective, it is a platform that allows us to coordinate the finances of the World Cups in a transparent way, and also allows users to help support the tournament in greater amounts outside of purchasing profile banners or subscribing to the osu!live Twitch channel. You can do any one (or all) of these things if you want, they'll all still go towards the tournament!
```
Comment (Walavouchey) [inline news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md:15]:
```suggestion
If you're still not familiar with Open Collective, it is a platform that allows us to coordinate the finances of the World Cups in a transparent way, and also allows users to help support the tournament in greater amounts outside of purchasing profile banners or subscribing to the osu!live Twitch channel. You can do any one (or all) of these things if you want and they'll still go towards the tournament!
```

File: news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+You can donate directly to the prize pool of the upcoming World Cup of your choosing in a lump sum with or without an Open Collective account, and you can even leave a little message of support that will be displayed beneath your contribution, if you like.
+
+*Please note that any "tips" made during the payment process go to Open Collective and not to us!* 
+
+For more details, refer to the [osu!taiko World Cup 2023 wiki article](/wiki/Tournaments/TWC/2023). You can discuss this event in the [official discussion thread](https://osu.ppy.sh/community/forums/topics/1722224) or at the public [community server](https://discord.gg/0Vxo9AsejDkGlk3H).
```
Comment (Walavouchey) [inline news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md:21]:
```suggestion
For more details, refer to the [osu!taiko World Cup 2023 wiki article](/wiki/Tournaments/TWC/2023). You can discuss this event in the [official discussion thread](https://osu.ppy.sh/community/forums/topics/1722224) or at the public [osu! community server](https://discord.gg/0Vxo9AsejDkGlk3H).
```

File: news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+| Round of 32 | Mar 25 & Mar 26 |
+| Round of 16 | Apr 01 & Apr 02 |
+| Quarterfinals | Apr 08 & Apr 09 |
+| Semifinals | Apr 15 & Apr 16 |
+| Finals | Apr 22 & Apr 23 |
+| Grand Finals | Apr 29 & Apr 30 |
+
+## Prizes
+
+The osu!taiko World Cup 2023 offers a $2,000 prize pool, along with limited edition merch! The prize pool may increase further from **[community donations](https://opencollective.com/osu-tournaments/events/osu-taiko-world-cup-2023-13130e56)**!
```
Comment (Walavouchey) [inline news/2023/2023-02-16-osu-taiko-world-cup-2023-registrations-now-open.md:39]:
```suggestion
The osu!taiko World Cup 2023 offers a $2,000 prize pool, along with limited-edition merch! The prize pool may increase further from **[community donations](https://opencollective.com/osu-tournaments/events/osu-taiko-world-cup-2023-13130e56)**!
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
otherwise ok

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**